### DISCUSSION TITLE
Warn: Gradient has outdated direction syntax

### DISCUSSION THREAD

**Discussion by DamianGlowala** [tailwindlabs/tailwindcss#12383] [Help] [answered]
I've been getting these warnings for a while. Does anyone know whether this is related to TailwindCSS and whether it could be somehow fixed on its end, even though warnings come from Vite?

![image](https://github.com/tailwindlabs/tailwindcss/assets/48835293/9a6636ab-36ab-48f2-b593-099a50561090)

**Comment by wongjn** [comment, marked as answer]
It seems like something outside of Tailwind core at least. The syntax in the warning seems to be related to radial gradients, but Tailwind does not ship with any. This would mean there is some custom configuration that may be invoking the warning message.

**Comment by DamianGlowala** [comment]
Thank you, will investigate further!
